plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/BaseProductTemplate.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Product Block Editor reusable template pieces.
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
+
+trait BaseProductTemplate {
+
+    /**
+     * Add all base template groups.
+     */
+    protected function add_base_template() {
+        $this->add_general_group();
+        $this->add_pricing_group();
+    }
+
+    /**
+     * Add general group and fields.
+     */
+    protected function add_general_group() {
+        $this->add_group(
+            array(
```

Comment:
Minor, but perhaps the array shorthand syntax (`[ 'foo' => 'bar' ]`) could be used instead? I think it would make these templates more readable.